modules/support_ticket/config/optional/field.field.support_ticket.ticket.field_assigned_to.yml

```diff
@@ -0,0 +1,30 @@
+uuid: 9d53bedd-ebbc-4d6a-ba32-50d2ddd45c83
```

Comment:
It's documented here: https://www.drupal.org/node/2087879  -- I'm just removing the uuid's.